### PR TITLE
Added Project Files

### PR DESCRIPTION
**Pull request by Divyan-H**
Team, please review my code and provide necessary suggestions for improvements. Your valuable input is highly appreciated. Thank you for your time and assistance.
Screenshot from 2023-07-11 15-56-41

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  <div class="dropdown">
+    <button class="dropbtn">Vision 
+    </button>
+    <div class="dropdown-content">
+      <a href="https://auroville.org/page/core-documents">3 Core Documents</a>
+      <a href="https://auroville.org/page/vision">Vision of the City</a>
+      <a href="https://auroville.org/page/matrimandir">Soul of the City</a>
+      <a href="https://auroville.org/page/integral-yoga">Intergral Yoga</a>
+      <a href="https://auroville.org/page/vision">The Founders</a>
+
```
Comment (sangeethailango) [inline index.html]: Indent the whole code properly

File: index.html [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+      <a href="https://auroville.org/category/green-practices">Green Practices</a>
+      <a href="https://auroville.org/page/health-and-wellness">Health & Wellness</a>
+      <a href="https://auroville.org/page/farming">Farming</a>
+      <a href="https://auroville.org/category/art-and-culture">Art & Culture</a>
+      <a href="https://auroville.org/category/media-and-communication">Media & Communication</a>
+      <a href="https://auroville.org/category/rural-development">Rural Development</a>
+      <a href="https://auroville.org/page/social-enterprises">Social Enterprises</a>
+      <a href="https://auroville.org/category/city-services">City Services</a>
+      </div>
+      </div>
```
Comment (sangeethailango) [inline index.html]: Dont know where this `div` is starting.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+        <div class="card">
+            <img src="https://files.auroville.org/auroville-org/7a5bbc77-5f7a-480a-84ec-7cdbb7e511a5.png" 
+            alt="support-auroville"
+            id="support-auroville">
+            <div class="container">
+                Support Auroville
+            </div>
+        </div>
+    </div>
+    <h1 id="overview">An Overview</h1>
```
Comment (sangeethailango) [inline index.html]: Use only one h1.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+                holding decades of experience
+            </p>
+        </div>
+    </a>
+    <a href="https://auroville.org/page/workshops-and-therapies">
+        <div class="cards">
+            <img src="https://files.auroville.org/auroville-org/e9671806-fe2b-4f52-806e-ee0f6b786030.jpg"
+            alt="Attend-workshop"
+            id="Attend-workshop"
+            style="width:100%;height:13em;">
```
Comment (sangeethailango) [inline index.html]: Don't use inline styling.

File: resources/css/my-index.css [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+}
+#footer a:hover {
+  color: rgb(255, 255, 255);
+  cursor: pointer;
+}
+#b {
+  padding-left: 10%;
+}
+#c {
+  padding-left: 10%;
```
Comment (sangeethailango) [inline resources/css/my-index.css:297]: Write meaningful names. if both #b and #c have same padding, why don't you use class?

**Review by sangeethailango — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (Divyan-H) [reply index.html]: ok done !

File: index.html (hunk 3 above)
Comment (Divyan-H) [reply index.html]: ok done !

File: resources/css/my-index.css (hunk 5 above)
Comment (Divyan-H) [reply resources/css/my-index.css:297]: ok done !

File: index.html (hunk 1 above)
Comment (Divyan-H) [reply index.html]: ok done !

File: index.html (hunk 2 above)
Comment (Divyan-H) [reply index.html]: ok done !

**Review by sakthibalan15 — APPROVED** [review]

**Pull request merged**